backend/sirius-web-diagrams/src/test/java/org/eclipse/sirius/web/diagrams/components/NodePositionProviderTestCases.java

```diff
@@ -39,77 +41,114 @@
 
     private static final Size DEFAULT_NODE_SIZE = Size.newSize().height(70).width(150).build();
 
-    private static final double STARTX = 20;
+    private static final double STARTX = 1000;
 
-    private static final double STARTY = 70;
+    private static final double STARTY = 1000;
 
     @Test
-    public void testNewNodesInDiagram() {
+    public void testNewNodesInClosedDiagram() {
```

Comment:
The distinction between opened/closed was unclear indeed.
What we want to distinct here is:

- create a single node, usually from an opened diagram. This leads to a recreation of the NodePositionProvider each time, as each creation leads to a separate refresh
- create several nodes at once, usually when creating nodes from the model explorer when the diagram is closed: on opening, a single refresh will create all the new nodes representations, using the same NodePositionProvider which will then keep a track of what has already been created to avoid overlaps

Would the following tests names be better:
testDiagramNewNodesOneByOne()
testDiagramSeveralNewNodesAtOnce()
testNodeNewNodesOneByOne()
testNodeSeveralNewNodesAtOnce()